src/coreclr-debug/activate.ts

```diff
@@ -36,16 +36,18 @@ export async function activate(thisExtension: vscode.Extension<CSharpExtensionEx
 
 async function checkForInvalidArchitecture(platformInformation: PlatformInformation, eventStream: EventStream): Promise<boolean> {
     if (platformInformation) {
-        if (platformInformation.isMacOS() && !CoreClrDebugUtil.isMacOSSupported()) {
-            eventStream.post(new DebuggerPrerequisiteFailure("[ERROR] The debugger cannot be installed. The debugger requires macOS 10.12 (Sierra) or newer."));
-            return true;
+        if (platformInformation.isMacOS()) {
+            // Support x86_64 and arm64
+            // Validate we are on compatiable macOS version if we are x86_64
+            if ((platformInformation.architecture !== "x86_64" && platformInformation.architecture !== "arm64") || 
+                (platformInformation.architecture === "x86_64" && !CoreClrDebugUtil.isMacOSSupported())) {
+                eventStream.post(new DebuggerPrerequisiteFailure("[ERROR] The debugger cannot be installed. The debugger requires macOS 10.12 (Sierra) or newer."));
+                return true;
+            }
         }
         else if (platformInformation.architecture !== "x86_64") {
             if (platformInformation.isWindows() && platformInformation.architecture === "x86") {
                 eventStream.post(new DebuggerPrerequisiteWarning(`[WARNING]: x86 Windows is not currently supported by the .NET Core debugger. Debugging will not be available.`));
-            } else if (platformInformation.isMacOS() && platformInformation.architecture === "arm64") {
-                eventStream.post(new DebuggerPrerequisiteWarning(`[WARNING]: arm64 macOS is not officially supported by the .NET Core debugger. You may experience unexpected issues when running in this configuration.`));
-                return false;
```

Comment:
Does the user still get a warning on arm64? I think we want one since:
1. The language service isn't downloaded/used I assume? (Could we just use the x64 one?)
2. It requires either the user install the preview of .NET 6 -or- modify their launch.json to target x64
3. The .NET runtime is still very much a preview